docs/doxygen/other/build_guide.dox.in

```diff
@@ -64,6 +63,9 @@ Other compilers may work, but are not supported.
 \subsection dep_fft gr-fft: Fast Frequency Transform
 \li fftw3f      (>= 3.0.1)   http://www.fftw.org/download.html
 
+\subsection dep_zeromq gr-zeromq: ZeroMQ Network layer
+\li ZeroMQ3     (>= 4.0.4)   http://www.zeromq.org/
```

Comment:
ZeroMQ3 but version >= 4.0.4 ?  Hmmm.
By the way, users will also need the C++ Bindings for 0MQ as well.
https://github.com/zeromq/cppzmq

Reply:
@awalls-cx18 yeah, was surprised by the versioning myself, but it's really ZMQ3 version 4.0.4. (the version is actually taken from personal notes, but it seems based on Ubuntu 14.04. So, I can't merge this without actually checking the version on CentOS 6 wouldn't also do).

Added cppzmq dependency. Since I don't know which version to pick (since they don't really have been versioning their files for very long), I found the version of zmq.hpp that's included in Ubuntu 14.04's libzmq3-dev package (which doesn't document it contains that file, nor where it downloaded it from, at which version), and threw a bit of git magic against the cppzmq repo to find the earliest matching revision

Reply:
Heck, no, since cppzmq mentions (as the only version it mentions) 3.3.0 to enable some features we don't need, I'll just go ahead and assume CentOS6/EPEL's 3.2.5 will do. This is risky, but whoever's building 3.7.12 on CentOS6 is probably not doing it for the first time, so I guess I'd be aware of bug reports relating to ZMQ version if that was problematic.

Reply:
And yes, I'm removing that 3 from the package name. It's a distro-specific thing that... at least debian and hence ubuntu ships ZMQ 4.0.4 in a package called libzmq3 . A distro trait I don't really like.